### PR TITLE
fix(flutter): update amplifyconfiguration.json to amplifyconfiguration.dart for flutter

### PR DESCRIPTION
**Pull request by Jordan-Nelson**
*Issue #, if available:* N/A

*Description of changes:*
- update `amplifyconfiguration.json` reference to `amplifyconfiguration.dart` for auth & analytics getting started tutorials
- update reference to `amplifyconfiguration.json`/`amplifyconfiguration.dart` in storage getting started tutorial to be targeted to flutter, ios, and android
- update `amplifyconfiguration.json` reference in analytics "record events" tutorial to `amplifyconfiguration.dart`

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/lib/storage/fragments/ios/getting-started/12_amplifyConfig.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Upon completion, `amplifyconfiguration.json` will be updated to reference provisioned S3 bucket.  Note that this file should already be a part of your project if you followed the [Project setup walkthrough](~/lib/project-setup/create-application.md).
```
Comment (richardmcclellan) [inline docs/lib/storage/fragments/ios/getting-started/12_amplifyConfig.md:1]:
```suggestion
Upon completion, `amplifyconfiguration.json` will be updated to reference a newly provisioned S3 bucket.  Note that this file should already be a part of your project if you followed the [Project setup walkthrough](~/lib/project-setup/create-application.md).
```

File: docs/lib/storage/fragments/flutter/getting-started/12_amplifyConfig.md [hunk 2]
```diff
@@ -0,0 +1 @@
+Upon completion, `amplifyconfiguration.dart` will be updated to reference provisioned S3 bucket.  Note that this file should already be a part of your project if you followed the [Project setup walkthrough](~/lib/project-setup/create-application.md).
```
Comment (richardmcclellan) [inline docs/lib/storage/fragments/flutter/getting-started/12_amplifyConfig.md:1]:
```suggestion
Upon completion, `amplifyconfiguration.dart` will be updated to reference a newly provisioned S3 bucket.  Note that this file should already be a part of your project if you followed the [Project setup walkthrough](~/lib/project-setup/create-application.md).
```

File: docs/lib/storage/fragments/android/getting-started/12_amplifyConfig.md [hunk 3]
```diff
@@ -0,0 +1 @@
+Upon completion, `amplifyconfiguration.json` will be updated to reference provisioned S3 bucket.  Note that this file should already be a part of your project if you followed the [Project setup walkthrough](~/lib/project-setup/create-application.md).
```
Comment (richardmcclellan) [inline docs/lib/storage/fragments/android/getting-started/12_amplifyConfig.md:1]:
```suggestion
Upon completion, `amplifyconfiguration.json` will be updated to reference a newly provisioned S3 bucket.  Note that this file should already be a part of your project if you followed the [Project setup walkthrough](~/lib/project-setup/create-application.md).
```

**Review by richardmcclellan — APPROVED** [review]
Minor grammar suggestions, but otherwise LGTM!

**Comment by medelman17** [comment]
Agree with @Jordan-Nelson re grammatical changes. Please incorporate above and squash all commits before merging.

**Review by richardmcclellan — APPROVED** [review]

**Review by drochetti — APPROVED** [review]

**Pull request merged**